fastlane/Fastfile

```diff
@@ -485,15 +489,26 @@ ENV["validate_translations"]="lintVanillaRelease"
     end
   end
 
-  desc "Import Gutenberg strings"
-  lane :localize_gutenberg do | options |
-    unless options[:skip_module_update] then
-      sh("git submodule init")
-      sh("git submodule update")
+  desc "Import strings from binary dependencies"
+  lane :localize_binary_deps do | options |
+    merge_strings_tool_file_path = File.join(ENV["PROJECT_ROOT_FOLDER"], 'tools/merge_strings_xml.py')
+    binary_imported_libraries.each do  | lib |
+      download_path = android_download_file_by_version(
+        library_name: lib[:name],
+        import_key: lib[:import_key],
+        repository: lib[:repository],
+        file_path: lib[:strings_file_path],
+        github_release_prefix: lib[:github_release_prefix])
+
+      if download_path.nil?
+        UI.error "Abort." unless UI.confirm("Can't download strings file for #{lib[:name]}. Do you want to continue?")
```

Comment:
Just using `error`, which doesn't exit the lane, is intentional?

I ended up in this path and the lane kept running even though it looked like it should stop. _There's another comment below about the request to commit the strings file event though it didn't change_.

![image](https://user-images.githubusercontent.com/1218433/110891761-c6220600-8346-11eb-8402-def66ca9dc64.png)

If that's not the intended behavior, maybe [`user_error!` or `shell_error!`](https://docs.fastlane.tools/advanced/actions/#user-input-and-output) could do the job.

Reply:
Right! I remember choosing it for a reason, but now I don't remember what I wanted to do 🤨 I've switched to `user_error` now